pyrene/src/components/TabView/TabView.jsx

```diff
@@ -189,11 +191,13 @@ TabView.propTypes = {
    */
   tabHeaderElement: PropTypes.element,
   /**
-   * Data input array for the tabs. Type: [{ name: string (required), renderCallback: func (required), disabled: bool }]
+   * Data input array for the tabs.
+   * Type: [{ name: string (required), nameRenderCallback: func, renderCallback: func (required), disabled: bool }]
    */
   tabs: PropTypes.arrayOf(PropTypes.shape({
     disabled: PropTypes.bool,
     name: PropTypes.string.isRequired,
+    nameRenderCallback: PropTypes.func,
```

Comment:
You could name that props, but it just from my perspective.

`iconRenderCallback: PropTypes.func,`

Reply:
@sosostris , I like your thoughts of always using `name`, it helps with consistency and prevents developers from misusing tabs for weird stuff.

Since the `name` is always present, `nameRenderCallback` can be somewhat confusing (it's not about the name, after all). `iconRenderCallback` is also not quite correct (because no one forces us to use an icon, could be anything).

`descriptionRenderCallback`?, also not quite correct. Of all alternatives I can think of, `nameRenderCallback` still makes most sense. 😝

Reply:
Hello @Sakai-san @neonnoon 😄  yes !!! I had great difficulty finding a proper name for it 😅  was also a bit concerned it might lead to some misunderstanding ... :( and once we merge the PR it would not be nice to come back to change again ...

The purpose of it is to display some additional quick information relevant to the content of the current tab, so `descriptionRenderCallback` actually better reflects this purpose. I also thought of `renderNameRightSection`,  `renderAuxiliaryInfo`, `additioanlInfoRenderCallback`, 😅  does it ring some bell?

Reply:
> renderAuxiliaryInfo

I like that one !